modules/cache-common/src/main/java/org/opensearch/cache/common/tier/TieredSpilloverCachePlugin.java

```diff
@@ -51,6 +51,11 @@ public List<Setting<?>> getSettings() {
             settingList.add(
                 TieredSpilloverCacheSettings.TIERED_SPILLOVER_DISK_STORE_NAME.getConcreteSettingForNamespace(cacheType.getSettingPrefix())
             );
+            settingList.add(
```

Comment:
Given the pluggable caches are experimental, I think we should put these settings behind feature flag as well so as to avoid these getting registered when feature flag is not enabled. Can be done in separate PR.